### PR TITLE
Added About Us

### PR DESCRIPTION
**Pull request by Abhishek-Punhani**
Added About Us modal

Fixes : #100

## Description
Added The About Us modal . Reused the structure used in rules modal 

## Checklist
- [ ] I have tested these changes locally.
- [ ] I have reviewed the code and ensured it follows the project's coding guidelines.
- [ ] I have updated the documentation, if necessary.
- [ ] I have assigned reviewers to this pull request.

## Screenshots (if applicable)
Screenshot 2024-06-14 233304
Screenshot 2024-06-14 233312

### CODE REVIEW

File: frontend/src/pages/Home.tsx [hunk 1]
```diff
@@ -53,9 +51,6 @@ const Home: React.FC = () => {
                 />
             </div>
             {showModal && <Modal onClose={() => setShowModal(false)} />}
-            {showRulesModal && (
```
Comment (kuv2707) [inline frontend/src/pages/Home.tsx:56]: These don't seem to be done by you.
Remove these

Comment (kuv2707) [reply frontend/src/pages/Home.tsx:56]: Also rebase once to make the deployment succeed

Comment (Abhishek-Punhani) [reply frontend/src/pages/Home.tsx:56]: Bhaiya voh maine hi remove kiya hai.....yeh already navbar mein covered hai so i removed it from home

**Comment by kuv2707** [comment]
Merged, thanks @Abhishek-Punhani

**Pull request merged**